infrastructure/smart-contracts/contracts/upgrades/node-license/NodeLicense8.sol

```diff
@@ -196,6 +196,19 @@ contract NodeLicense8 is ERC721EnumerableUpgradeable, AccessControlUpgradeable
      * @param _promoCode The promo code.
      */
     function mint(uint256 _amount, string calldata _promoCode) public payable {
+        _mintInternal(msg.sender, _amount, _promoCode);
+    }
+    /**
+     * @notice Mints new NodeLicense tokens.
+     * @param _mintToAddress The address to mint the tokens to.
+     * @param _amount The amount of tokens to mint.
+     * @param _promoCode The promo code.
+     */
+    function mintTo(address _mintToAddress, uint256 _amount, string calldata _promoCode) public payable {
```

Comment:
We should double check the function name doesn't need to be `mint` instead of `mintTo` since this change is for a third party integration.

Reply:
I agree, I actually tried to just use `mint` with a different function signature and got errors because crossmint couldn't seem to differentiate between them, so I changed to mintTo and was able to get it to work. I have tested and confirmed that it works with crossmint. 👍